.gitignore

```diff
@@ -8,6 +8,7 @@ build
 !**/src/**/build
 
 # User-specific stuff
+.idea/
```

Comment:
What are you trying to exclude here? We do have some files in .idea that we want tracked that are already in the repository.

Reply:
Yea that makes sense. I'll exclude the pesky files specifically: they are `.idea/aws.xml` and `.idea/kotlinc.xml`